packages/botonic-plugin-contentful/src/search/search-by-keywords.ts

```diff
@@ -35,11 +38,8 @@ export class SearchByKeywords {
     )
     const results = kws.findCandidatesWithKeywordsAt(inputText)
     return results.map(res => {
-      const candidate = res.candidate as SearchResult
-      // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
-      // @ts-ignore
-      candidate.match = res.match
-      return candidate
+      const score = res.match.length / inputText.raw.length
```

Comment:
Can `inputText.raw.length` be `0`?

Reply:
:+1: very good point. Actually I think we get strange results somewhere else if we get empty texts.I added a check